<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confirmDialog</class>
 <widget class="QDialog" name="confirmDialog">
  <property name="geometry">
   <rect>
	<x>0</x>
	<y>0</y>
	<width>400</width>
	<height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Execute Copy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
	<number>5</number>
   </property>
   <property name="topMargin">
	<number>5</number>
   </property>
   <property name="rightMargin">
	<number>5</number>
   </property>
   <property name="bottomMargin">
	<number>5</number>
   </property>
   <item>
	<widget class="QLabel" name="label_Src">
	 <property name="font">
	  <font>
	   <pointsize>12</pointsize>
	  </font>
	 </property>
	 <property name="text">
	  <string>Source</string>
	 </property>
	</widget>
   </item>
   <item>
	<widget class="QTextEdit" name="textEdit_Src">
	 <property name="font">
	  <font>
	   <pointsize>11</pointsize>
	  </font>
	 </property>
	 <property name="textInteractionFlags">
	  <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
	 </property>
	</widget>
   </item>
   <item>
	<widget class="QLabel" name="label_Dst">
	 <property name="font">
	  <font>
	   <pointsize>12</pointsize>
	  </font>
	 </property>
	 <property name="text">
	  <string>DestDir</string>
	 </property>
	</widget>
   </item>
   <item>
	<widget class="QTextEdit" name="textEdit_Dst">
	 <property name="sizePolicy">
	  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
	   <horstretch>0</horstretch>
	   <verstretch>0</verstretch>
	  </sizepolicy>
	 </property>
	 <property name="maximumSize">
	  <size>
	   <width>16777215</width>
	   <height>75</height>
	  </size>
	 </property>
	 <property name="font">
	  <font>
	   <pointsize>11</pointsize>
	  </font>
	 </property>
	 <property name="textInteractionFlags">
	  <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
	 </property>
	</widget>
   </item>
   <item>
	<widget class="QDialogButtonBox" name="buttonBox">
	 <property name="layoutDirection">
	  <enum>Qt::LeftToRight</enum>
	 </property>
	 <property name="orientation">
	  <enum>Qt::Horizontal</enum>
	 </property>
	 <property name="standardButtons">
	  <set>QDialogButtonBox::Cancel</set>
	 </property>
	 <property name="centerButtons">
	  <bool>true</bool>
	 </property>
	</widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>confirmDialog</receiver>
   <slot>accept()</slot>
   <hints>
	<hint type="sourcelabel">
	 <x>248</x>
	 <y>254</y>
	</hint>
	<hint type="destinationlabel">
	 <x>157</x>
	 <y>274</y>
	</hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>confirmDialog</receiver>
   <slot>reject()</slot>
   <hints>
	<hint type="sourcelabel">
	 <x>316</x>
	 <y>260</y>
	</hint>
	<hint type="destinationlabel">
	 <x>286</x>
	 <y>274</y>
	</hint>
   </hints>
  </connection>
 </connections>
</ui>
